Resources/Meta Analysis/qml/RobustBayesianMetaAnalysis.qml

```diff
@@ -28,6 +28,16 @@ Form
 		name:					"measures"
 		radioButtonsOnSameRow:	true
 		columns:				2
+		onValueChanged:	if(measures_correlation.checked) {
+			advanced_mu_transform.value				= "cohens_d"
+			// TODO: make this work >>
+			advanced_mu_transform_log_OR.enabled	= true
+			advanced_mu_transform_fishers_z.enabled	= false
+		} else if (measures_OR.checked){
+			advanced_mu_transform.value				= "log_OR"
+			advanced_mu_transform_log_OR.enabled	= false
+			advanced_mu_transform_fishers_z.enabled	= true
+		}
```

Comment:
Maybe @boutinb  could help?
I manage to change the default value of the at line 1852 using the line 32 and 37, but I can't figure out how to disable the "log_OR" option when correlations are selected and "fishers_z" option when "OR" are selected. 

I also tried setting the values and value directly here - the correct value and options are passed, but the dropdown box is missing labels:
```
onValueChanged:	if(measures_correlation.checked) {
			advanced_mu_transform.values = 	
			[
				{ label: qsTr("Cohen's d"),		value: "cohens_d"},
				{ label: qsTr("Fisher's z"),	value: "fishers_z"}
			]
			advanced_mu_transform.value  = "cohens_d"
		} else if (measures_OR.checked){
			advanced_mu_transform.values	 = 	
			[
				{ label: qsTr("log(OR)"),		value: "log_OR"},
				{ label: qsTr("Cohen's d"),		value: "cohens_d"}
			]
			advanced_mu_transform.value  = "log_OR"
		}
```